package.json

```diff
@@ -64,6 +64,7 @@
 		"@stripe/stripe-js": "^1.20.3",
 		"broadcastchannel-polyfill": "1.0.1",
 		"chalk": "^4.1.2",
+		"core-js": "^3.19.0",
```

Comment:
- let's not add this `core-js` dependency if we can avoid it

Reply:
removed cors and added svg inline
Svg is optimized